Update Ubuntu-Development-Setup.rst

m-elwin opened the pull request:
vcs tool can directly download the .repos file, so I combined the wget/vcs commands into one.

m-elwin committed:
Update Ubuntu-Development-Setup.rst

vcs tool can directly download the .repos file, so I combined the wget/vcs commands into one.

clalancette reviewed (changes requested):
The less number of steps that users have to cut-n-paste, the better.  So I'm for this change.

That said, I think our instructions should be consistent.  So with that in mind, I think we'll also want to update the following:

* https://github.com/ros2/ros2_documentation/blob/5a59c4c7128b77c1c384a128658da3544f6a80c6/source/How-To-Guides/RQt-Source-Install.rst?plain=1#L38-L39
* https://github.com/ros2/ros2_documentation/blob/5a59c4c7128b77c1c384a128658da3544f6a80c6/source/Installation/Alternatives/Windows-Development-Setup.rst?plain=1#L75-L105
* https://github.com/ros2/ros2_documentation/blob/5a59c4c7128b77c1c384a128658da3544f6a80c6/source/Installation/Alternatives/macOS-Development-Setup.rst?plain=1#L150-L151
* https://github.com/ros2/ros2_documentation/blob/5a59c4c7128b77c1c384a128658da3544f6a80c6/source/Installation/Alternatives/RHEL-Development-Setup.rst?plain=1#L87-L88

m-elwin committed:
Updated to use the --input option of vcs import

This eliminates the need to dowload wget/curl for some instructions
It also removes the need for separate instructions for windows bash/power shell

m-elwin commented:
That makes sense @clalancette. I made some  changes, and it especially simplified the windows alternative instructions I think, but I'm not able to test them on windows.

clalancette reviewed (approved):
This looks fantastic, thanks for the improvement!
